configuration/multimedia.md

```diff
@@ -57,58 +54,42 @@ Import GPG-Key for spotify
 As the template VM can't connect to internet you need to get the public key file from another AppVM and copy it to the template VM. The easiest way is to use the Qubes Clipboard to copy the keys from the AppVM where you get the key to the Template VM.
 
 In an AppVM which has Internet access:
-- Open <https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xEFDC8610341D9410>
+- Open https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xA87FF9DF48BF1C90
 - Copy content of page to the Clipboard (Ctrl+A and Ctrl+C)
 - open a Terminal in this AppVM and copy the content of the clipboard to a file
   `xclip -o > spotify.pubkey`
 
-Copy the public signing key over to the multimedia template VM
-- copy the file via `qvm-copy-to-vm t-multimedia spotify.pubkey`
-- or create a new file on the Template VM and copy the content of the clipboard (the public key)
-  Copy content of page to the Qubes Clipboard (Ctrl+C and then Shift+Ctrl+C)
-  Switch to the gnome terminal in the Multimedia Template VM
-  `nano spotify.pubkey`
-  Paste the content from the Qubes Clipboard into nano (Shift+Ctrl+V and then Paste)
-  Save the file (Ctrl+O <Enter> Ctrl+X)
+Verify Fingerprint
 
-Check the signature of the signing key (in the multimedia Template VM).
+Check the signature of the signing key.
 Hint: depending on your installed version of GnuPG the command to show a public might slightly be different.
 See [this StackExchange question](https://unix.stackexchange.com/questions/391344/gnupg-command-to-show-key-info-from-file) for more information.
 If this command doesn't show a fingerprint choose one of the other commands mentioned in the above link.
 
 `gpg --with-fingerprint spotify.pubkey`
 
-This should look like:
-
-    [user@t-multimedia ~]$ `gpg --with-fingerprint spotify.pubkey`
-
-    pub  4096R/341D9410 2017-07-25 Spotify Public Repository Signing Key <tux@spotify.com>
-
-         Key fingerprint = 0DF7 31E4 5CE2 4F27 EEEB  1450 EFDC 8610 341D 9410
-
+Compare it for example to https://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0xA87FF9DF48BF1C90&fingerprint=on
```

Comment:
Instead of verification with another mean of downloading the key based on its ID, better verify if that key really belong to spotify. For example here: https://www.spotify.com/de/download/linux/

Reply:
I agree, I'll add this to the howto, as I have also planned to rewrite it, to make it easier to follow.

Reply:
The long one 932..C90 is a fingerprint. Key ID (in gpg at least) is simply last 8/16 chars of the fingerprint.
For key verification, you need something that externally confirm given key really belong to spotify. Keyserver on its own doesn't give it to you - it only decode information contained in the key and anyone can generate key with the same display name, but not the fingerprint.
Anyway, spotify website pointed above do contain key fingerprint, so have extra info: someone controlling https://www.spotify.com certify that this key really belongs to spotify.